README.md

```diff
@@ -568,6 +573,10 @@ https://developer.mozilla.org/en-US/docs/Web/API
 - .getElementsByTagName()
 - .querySelector()
 - .querySelectorAll()
+
+##4.13 Concurrency model and Event Loop
+- https://developer.mozilla.org/en-US/docs/Web/JavaScript/EventLoop
+- https://youtu.be/8aGhZQkoFbQ
```

Comment:
This is a text-only list, videos can be added in another separated list
EventLoop is part of JavaScript, not Web APIs